examples/Python/PET/interactive/03-basic_reconstruction.py

```diff
@@ -69,13 +69,14 @@
 
 #%% display of image
 reconstructed_array=reconstructed_image.as_array()
-slice=reconstructed_array.shape[0]//3;
-show_2D_array('reconstructed image after 5 sub-iterations',reconstructed_array[slice,:,:,]);
+# slice=reconstructed_array.shape[0]//3;
+sliceno = 9
```

Comment:
I'd prefer to have it in terms of the `shape` somehow, but fine to leave if that's too hard

Reply:
> I have the same comments as @evgueni-ovtchinnikov
> 
> * rename `sliceno` to `slice_num` etc

OK

> * why remove `help`?

I removed `help` thinking that it may fix the problem that that was the reason spyder wouldn't run the whole script correctly. As that is not the case I'll put it back. However, the problem of spyder reaching the end without running correctly still remains.